network.ts

```diff
@@ -0,0 +1,43 @@
+export interface TablelandNetworkConfig {
```

Comment:
Yeh nice, I do think this makes _using_ this downstream a bit easier. Additionally, we can pull in this interface and use that so we know we aren't trying to support an unsupported chain.